nh_clinical/security/ir.model.access.csv

```diff
@@ -43,7 +43,7 @@ admin_group_access_nh_clinical_user_responsibility_allocation_access,admin:acces
 admin_group_access_nh_clinical_user_management,admin:access_nh_clinical_user_management,model_nh_clinical_user_management,nh_clinical.group_nhc_admin,1,1,1,0
 ,,,,,,,
 dev_group_access_nh_clinical_pos,admin:access_nh_clinical_pos,model_nh_clinical_pos,nh_clinical.group_nhc_dev,1,1,1,1
-dev_group_access_nh_clinical_location,admin:access_nh_clinical_location,model_nh_clinical_location,nh_clinical.group_nhc_dev,1,1,1,1
+dev_group_access_nh_clinical_location,admin:access_nh_clinical_location,model_nh_clinical_location,nh_clinical.group_nhc_dev,1,1,1,0
```

Comment:
This is out of scope for this issue. Please raise ticket to deal with it at a later date, once it's properly defined.

Reply:
Raised question [EOBS-400 What should permissions for the location model be?](https://jupiter.bjss.com/browse/EOBS-2400).